frontend/views/components/ListItem.vue

```diff
@@ -76,31 +67,39 @@ export default {
 .c-item-link {
   display: flex;
   align-items: center;
-  padding: 0 $spacer;
-  height: $spacer-lg;
+  padding: 0 1.25rem;
+  height: 3rem;
   cursor: pointer;
   transition: background-color ease-out 0.3s;
+  font-family: "Poppins";
   color: $text_0;
+  font-weight: 100;
+  position: relative;
 
   i {
-    width: 1rem;
-    margin-right: $spacer-md;
-    font-size: 0.77rem;
-    color: $general_0;
+    width: $spacer;
+    margin-right: 1.2rem;
```

Comment:
Based on the last Monday call, should this value be 1rem to stick with the current spacing values? (I checked Figma and it's 16px indeed, not 19px)